packages/sdk/contractkit/src/wrappers/FeeCurrencyDirectoryWrapper.ts

```diff
@@ -17,5 +17,9 @@ export class FeeCurrencyDirectoryWrapper extends AbstractFeeCurrencyWrapper<FeeC
     return this.getCurrencies()
   }
 
-  // TODO add other methods as well
+  getExchangeRate: (token: StrongAddress) => Promise<{ numerator: string; denominator: string }> =
+    proxyCall(this.contract.methods.getExchangeRate)
```

Comment:
dont forget to pass the token param!

Reply:
as discussed not needed, but will add coverage